scratch-gui/src/addons/addons/editor-searchable-dropdowns/userscript.js

```
@@ -124,6 +124,22 @@ export default async function ({ addon, console, msg }) {
     resultOfLastGetOptions = options;
     return options;
   };
+  
+  const oldFieldTextDropdownGetOptions = Blockly.FieldTextDropdown.prototype.getOptions;
```

ESLint

[error] 'oldFieldTextDropdownGetOptions' is assigned a value but never used. Allowed unused vars must match /^_/u.

(no-unused-vars)